patches/api/0480-Expose-server-build-information.patch

```diff
@@ -0,0 +1,257 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: masmc05 <masmc05@gmail.com>
+Date: Tue, 14 May 2024 21:38:49 -0700
+Subject: [PATCH] Expose server build information
+
+Co-authored-by: Riley Park <rileysebastianpark@gmail.com>
+
+diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..3ba385138c49d6bcc7a1e4833cb1e2b48af32373
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
+@@ -0,0 +1,121 @@
++package io.papermc.paper;
++
++import java.time.Instant;
++import java.util.Optional;
++import java.util.OptionalInt;
++import net.kyori.adventure.key.Key;
++import net.kyori.adventure.util.Services;
++import org.jetbrains.annotations.ApiStatus;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Information about the current server build.
++ */
++@ApiStatus.NonExtendable
++public interface ServerBuildInfo {
++    /**
++     * The brand id for Paper.
++     */
++    Key BRAND_PAPER_ID = Key.key("papermc", "paper");
++
++    /**
++     * Gets the {@code ServerBuildInfo}.
++     *
++     * @return the {@code ServerBuildInfo}
++     */
++    static @NotNull ServerBuildInfo get() {
++        //<editor-fold defaultstate="collapsed" desc="Holder">
++        final class Holder {
++            static final ServerBuildInfo INSTANCE = Services.service(ServerBuildInfo.class).orElseThrow();
++        }
++        //</editor-fold>
```

Comment:
Did we confirm that simply classloading ServerBuildInfo doesn't classload the method local class and throw an exception if no service exists?